CHANGES.md

```diff
@@ -1,5 +1,7 @@
 # git master
 
+* [486](https://github.com/Eyescale/CMake/pull/486):
+  Fix coverage report generation for non-subprojects
```

Comment:
Indeed, you have solved it for the subproject case. Don't you mean "for non-superprojects"?